app/scenes/Document/components/Comments.tsx

```diff
@@ -132,10 +186,32 @@ const NoComments = styled(Flex)`
 `;
 
 const Wrapper = styled.div<{ $hasComments: boolean }>`
-  padding-bottom: ${(props) => (props.$hasComments ? "50vh" : "0")};
   height: ${(props) => (props.$hasComments ? "auto" : "100%")};
 `;
 
+const ScrollToRecentSize = "32px";
+
+const ScrollToRecent = styled.div`
```

Comment:
Could you use the existing `ButtonSmall` component and just make it rounded? Could use the text `New comments` perhaps

Reply:
Converted.

(_and wow, the translation script seems to be able to figure out where to inject the new string in the json :+1:_ )